### PR TITLE
Add WRLinux product WRLinux8 and WRLinux1019 support

### PR DESCRIPTION
**Pull request by dehuo0**
Description:
Rename wrlinux to wrlinux8 to specify the Wind River Linux product version number.
Since there are serval version Wind River products start to support SCAP e.g. WRLinux8,
 WRLinux1019, the supported profiles and functions would be different between these 
version products. The old source code for wrlinux in SCAP is just for WRLinux8, so rename 
the wrlinux to wrlinux8.
Add the WRLinux1019 platform support, provide the relevant functional files for the WRLinux1019
e.g. profiles, cpe directory etc.

Rationale:
Add home directories for WRLinux8 and WRLinux1019;
Add multi_platform_wrlinux platform to shared.xml files for wrlinux supported rules;
Add wrlinux prodtype to rule.yml files for wrlinux supported rules;
Add multi_platform_wrlinux platform to remediate bash scripts for wrlinux supported rules;

### CODE REVIEW

**Comment by yuumasato** [comment]
@openscap-ci ok to test

File: shared/checks/oval/installed_OS_is_wrlinux1019.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <criterion comment="WRLinux is installed" test_ref="test_wrlinux1019" />
+    </criteria>
+  </definition>
+  
+<ind:textfilecontent54_test check="all" check_existence="at_least_one_exists" comment="Check WRLinux10.19 version" id="test_wrlinux1019" version="1">
+    <ind:object object_ref="obj_name_wrlinux1019" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_name_wrlinux1019" version="1" comment="Check WRLinux1019 produce name">
+    <ind:filepath>/etc/os-release</ind:filepath>
+    <ind:pattern operation="pattern match">^NAME=.Wind[\s]+River[\s]+Linux</ind:pattern>
```
Comment (yuumasato) [inline shared/checks/oval/installed_OS_is_wrlinux1019.xml]: I don't know what is included in `/etc/os-release`, but if possible, I recommend to improve the pattern to match only WRLinux 1019.
Something like this: https://github.com/ComplianceAsCode/content/blob/69ae8f5a1c696b05beb5f1ab5ccdd5d40e90dac8/shared/checks/oval/installed_OS_is_debian8.xml#L30
or https://github.com/ComplianceAsCode/content/blob/69ae8f5a1c696b05beb5f1ab5ccdd5d40e90dac8/shared/checks/oval/installed_OS_is_ubuntu1604.xml#L22

File: shared/checks/oval/installed_OS_is_wrlinux8.xml [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+  <ind:family_object id="obj_unix_wrlinux8" version="1" />
 
-  <file_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" check_existence="at_least_one_exists" version="1" id="test_wrlinux" check="all" comment="Test presence of /etc/wrlinux-release.">
-      <object object_ref="obj_test_wrlinux" />
+  <file_test xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" check_existence="at_least_one_exists" version="1" id="test_wrlinux8" check="all" comment="Test presence of /etc/wrlinux-release.">
+      <object object_ref="obj_test_wrlinux8" />
   </file_test>
 
-  <file_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" version="1" id="obj_test_wrlinux" >
+  <file_object xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" version="1" id="obj_test_wrlinux8" >
```
Comment (yuumasato) [inline shared/checks/oval/installed_OS_is_wrlinux8.xml]: Is the file `/etc/wrlinux-release` exclusive to WRLinux8?
I recommend to make the check more specific for version 8.

File: CMakeLists.txt [hunk 3]
```diff
@@ -80,7 +80,8 @@ option(SSG_PRODUCT_SLE12 "If enabled, the SLE12 SCAP content will be built" ${SS
 option(SSG_PRODUCT_UBUNTU1404 "If enabled, the Ubuntu 14.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_UBUNTU1604 "If enabled, the Ubuntu 16.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_UBUNTU1804 "If enabled, the Ubuntu 18.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
-option(SSG_PRODUCT_WRLINUX "If enabled, the WRLinux SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
+option(SSG_PRODUCT_WRLINUX8 "If enabled, the WRLinux SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
+option(SSG_PRODUCT_WRLINUX1019 "If enabled, the WRLinux SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
```
[the remark this answers is not in the document]
Comment (dehuo0) [reply CMakeLists.txt]: OK I will update this and submit another version pull request.

File: shared/checks/oval/installed_OS_is_wrlinux1019.xml (hunk 1 above)
Comment (dehuo0) [reply shared/checks/oval/installed_OS_is_wrlinux1019.xml]: WRLinux1019 use /etc/os-release to record the relevant version message and produce name, I will match more message in /etc/os-release file for WRLinux1019, and submit another version pull request.

File: shared/checks/oval/installed_OS_is_wrlinux8.xml (hunk 2 above)
Comment (dehuo0) [reply shared/checks/oval/installed_OS_is_wrlinux8.xml]: Thanks for the remind.
WRLinux8 use /etc/wrlinux-release to record the relevant version message but WRLinux1019 does not use it any more. I will check more version message in /etc/wrlinux-release file for WRLinux8, and submit another version pull request

**Comment by yuumasato** [comment]
@dehuo0 Next time, there is no need to open a new PR.
Just make new commits and push to the same branch, this way it will also be easier to review.

Closing in favor of #4594

**Pull request closed without merging**